west.yml

```diff
@@ -104,7 +104,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: fd79b800bddf63b0a212a70007a6b612226e134c
+      revision: pull/593/head
```

Comment:
SHA: b0a356db59463e7f16f59fea5ca9a7f73c1b6ae1

Reply:
Thanks, just updated this with the correct SHA.